modules/motions/ui/MotionFormStartNew/Parts/StartSDVTNodeOperatorsAdd.tsx

```diff
@@ -56,11 +61,10 @@ export const formParts = ({
   createMotionFormPart({
     motionType: ALLOWED_SDVT_NODE_OPERATORS_ADD_MAP[registryType].motionType,
```

Comment:
Do we really need to use `ALLOWED_SDVT_NODE_OPERATORS_ADD_MAP`? I saw that structure in other forms but it is used there to combine multiple factories with different types in one form. In this form, we don't have any other factories that we could combine, so it could be omitted. We can just use `SDVTNodeOperatorsAdd` as motion type and omit using `registryType` in this component.